examples/shopify/lib/config.ts

```diff
@@ -1,30 +1,58 @@
-export type Config = {
+type PublicConfig = {
   shopify: {
     storeName: string
     accessToken: string
   }
+}
+
+type PrivateConfig = {
   makeswift: {
     siteApiKey: string
     productTemplatePathname: string
   }
 }
 
-function getEnvVarOrThrow(key: string): string {
-  const value = process.env[key]
+export type Config = PublicConfig & PrivateConfig
 
+/**
+ * public env values must be inlined for the value to be exposed to the browser
+ * https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser
+ * @param key -> name of env variable
```

Comment:
There's a specific JSDoc syntax for defining parameters: https://jsdoc.app/tags-param.html

Don't know if you were going for that, but if you were, you might want to avoid the `->`.

Reply:
because I am now using tiny-invariant this comment is no longer needed / I removed it